lib/rouge/lexers/ttcn3.rb

```diff
@@ -0,0 +1,129 @@
+# -*- coding: utf-8 -*- #
+# frozen_string_literal: true
+
+module Rouge
+  module Lexers
+    class TTCN3 < RegexLexer
+      title "TTCN3"
+      desc "The TTCN3 programming language (ttcn-3.org)"
```

Comment:
I've reformatted the description to match the style of most other lexers in the repo.